Mail/Common.pm

```diff
@@ -184,10 +185,13 @@ sub set_recipients {
 sub test_language_setup {
     my($self) = @_;
     b_use('IO.Config')->introduce_values({
-	'Bivio::Mail::Common' => {
-            allow_resend_from => [],
-	},
-    }) if @{$self->internal_get_config->{allow_resend_from}};
+        'Bivio::Mail::Common' => {
+            in_btest => 1,
+            @{$self->internal_get_config->{allow_resend_from}} ? (
```

Comment:
It seems like you could just set `allow_resend_from => []` without the conditional, that is, it would always override to be empty which seems logical.

Reply:
Hmm, I don't think that override would have the intended effect anyway as the only relevant place (https://github.com/biviosoftware/perl-Bivio/blob/50-fix-btest-rewrite-from/Mail/Outgoing.pm#L406) checks `allow_resend_from_re`, not `allow_resend_from`. That config override is further obviated by `in_btest`.

Reply:
Ah, looks like you're right wrt `allow_resend_from_re` getting recomputed. However, I think the override to clear `allow_resend_from` should be removed anyway as it appears that was only being done before so that the code would consider `rewrite_from_domains` instead (which used to be overridden to `rewrite_from_domains => ['avoid-all-rewrites-in-mock-sendmail']`). So, even if `in_btest` didn't exist, the override to clear `allow_resend_from` wouldn't be correct.